### ISSUE TITLE
peview.exe 3.0.12105.7578 crash

### ISSUE THREAD

**Issue by urielmann** [winsiderss/systeminformer#2041]
### Brief description of your issue

Starting peview.exe 3.0.12105.7578 on Windows 10 VMWare VM result in a crash
[peview.dmp](https://github.com/winsiderss/systeminformer/files/15101197/peview.dmp)
![image](https://github.com/winsiderss/systeminformer/assets/56919585/53ad3f10-a7a1-4f1d-8158-49a8cf620397)

**Comment by dmex** [comment]
> 10.0.19041.1

The stack shows a bug in the OS and your version of Windows was last updated 5 years ago?

**Comment by MagicAndre1981** [comment]
>  your version of Windows was last updated 5 years ago?

no, Windbg also shows this for me on in dmp files from Windows 10 22H2 19045 with latest patches.
